### DISCUSSION TITLE
Database restoration error

### DISCUSSION THREAD

**Discussion by reubendrummond** [supabase/supabase#26422] [Questions] [answered]
Hey there. We've run into the following error when attempting to restore a db backup on the Supabase portal:

Failed to restore from backup: All subscriptions and replication slots must be dropped before a backup can be restored.

Couldn't find any docs on this so any help would be appreciated!

**Comment by GaryAustin1** [comment, marked as answer]
Searched on replication slots here...
https://github.com/orgs/supabase/discussions/21830
